docroot/themes/humsci/su_humsci_admin/package-lock.json

```diff
@@ -7,7 +7,7 @@
     "abbrev": {
```

Comment:
Are you sure you want changes to this file in this PR? This seems outside of the scope of the PR.

Reply:
i'm guessing that was injected by npm during `npm install` and the latest npm version